src/locales/en_US.js

```diff
@@ -677,10 +677,11 @@ const strings = {
   fio_address_details_screen_alert_button: 'View Details',
   fio_address_details_screen_registered: 'Registered',
   fio_address_details_screen_expires: 'Expires',
-  fio_address_details_screen_bundles: 'Remaining bundles',
+  fio_address_details_screen_bundles: 'Remaining bundled transactions',
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name